### PR TITLE
:sparkles: feat(*): new config that support no emojis

### PR DESCRIPTION
**Pull request by jimmyfortinx**
Since we don't want to force the usage of emojis in our projects we need to have two commitizen configurations. To not introduce breaking changes, I kept the emojis version as the default one and introduced a new `no-emojis` config.

The default once still can be used like that:

```json
  "config": {
    "commitizen": {
      "path": "./node_modules/cz-ovh-commit"
    }
  }
```

And the new `no-emojis` version can be used like that:

```json
  "config": {
    "commitizen": {
      "path": "./node_modules/cz-ovh-commit/no-emojis"
    }
  }
```

### CODE REVIEW

File: configs/ovh-config-no-emojis.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    // should prompt about the body
+    useBody: ["feat", "chore", "style", "test"],
+
+    // should prompt about the footer which indicates bug fix
+    useFooter: ["feat", "fix"],
+
+    // first line max header lenght
+    headerLength: 72,
+
+    // parser for the commit msg, the only value guarented is type
```
Comment (mlarose) [inline configs/ovh-config-no-emojis.js:35]: guaranteed

File: configs/ovh-config-no-emojis.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    // should prompt about the header
+    useHeader: ["feat", "chore", "style", "test", "docs", "refactor", "merge"],
+
+    // should prompt about the body
+    useBody: ["feat", "chore", "style", "test"],
+
+    // should prompt about the footer which indicates bug fix
+    useFooter: ["feat", "fix"],
+
+    // first line max header lenght
```
Comment (mlarose) [inline configs/ovh-config-no-emojis.js:32]: length

File: configurable-prompter.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
+"use strict";
+
+const inquirer = require("inquirer");
+const _ = require("lodash");
+const findConfig = require("find-config");
+
+function getConfig (filename) {
+    const pkg = findConfig.require("package.json");
+    const configFile = _.get(pkg, "config.cz-ovh-commit.config", filename);
+    console.log("Using config file: ", configFile);
```
Comment (mlarose) [inline configurable-prompter.js:10]: remove console.log?

File: configurable-prompter.js [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+                        message: "Please denote the SCOPE of the commit: (optional)",
+                        when: (answers) => _.includes(config.useScope, answers.type),
+                        filter: (input) => _.trim(input)
+                    },
+                    {
+                        type: "input",
+                        name: "header",
+                        message: "Please write a HEADER: (mandatory)",
+                        when: (answers) => _.includes(config.useHeader, answers.type),
+                        validate: (input) => input && _.trim(input).length < config.headerLength && _.trim(input).length !== 0 ? true : `Message should not be empty and should be shorter than ${config.headerLength} characters`,
```
Comment (mlarose) [inline configurable-prompter.js:48]: could be simplified (an empty string evaluates to false) and lisibility could be improved. this line is too long

File: configurable-prompter.js [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+                        filter: (input) => _.trim(input, [" ."])
+                    },
+                    {
+                        type: "input",
+                        name: "body",
+                        message: "Please write a BODY: (optional)",
+                        when: (answers) => _.includes(config.useBody, answers.type),
+                        filter: (input) => _.trim(input)
```
Comment (mlarose) [inline configurable-prompter.js:56]: filter: _.trim would do

File: configurable-prompter.js [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+
+const inquirer = require("inquirer");
+const _ = require("lodash");
+const findConfig = require("find-config");
+
+function getConfig (filename) {
+    const pkg = findConfig.require("package.json");
+    const configFile = _.get(pkg, "config.cz-ovh-commit.config", filename);
+    console.log("Using config file: ", configFile);
+    return Object.assign({}, require(filename), require(configFile));
```
Comment (mlarose) [inline configurable-prompter.js:11]: be mindful that Object.assign does not perform deep merging.

Comment (mlarose) [reply configurable-prompter.js:11]: also both requires are not guarded. are the two files expected to always exist?

**Review by mlarose — APPROVED** [review]
minor points of attention, typos and such

**Comment by antleblanc** [comment]
I close it for inactivity, but feel free to re-open it if needed.

**Pull request closed without merging**